cyences_app_for_splunk/appserver/static/cs_configuration.js

```diff
@@ -331,4 +331,64 @@ require([
     $(`#malicious_button`).on("click", function(){
         updateMaliciousIPConfiguration();
     });
+
+    function getSophosConfiguration(){
```

Comment:
Keep name everywhere as `SophosEndpoint`
so, when you add `SophosFirewall` configuration it would be easy to distinguish.